combustion-android-ble/src/main/java/inc/combustion/framework/ble/ProbeManager.kt

```diff
@@ -148,7 +148,14 @@ internal open class ProbeManager (
         addJob(owner.lifecycleScope.launch(Dispatchers.IO) {
             while(isActive) {
                 if(isConnected.get() && mac != SimulatedProbeManager.SIMULATED_MAC) {
-                    remoteRssi.set(peripheral.rssi())
+                    try {
+                        remoteRssi.set(peripheral.rssi())
+                    } catch (e: Exception) {
+                        if (DebugSettings.DEBUG_LOG_BLE_OPERATIONS) {
+                            Log.d(LOG_TAG,
+                                "Ignoring exception reading remote RSSI: \n${e.stackTrace}")
+                        }
+                    }
```

Comment:
What kind of exceptions were you seeing here? Is `debug` the right level of logging? Is there one that we can explicitly ignore? If so, it might make sense to catch that type of exception rather than the generic `Exception`.

Reply:
I noticed BLE disconnect exceptions while the RSSI read operation was in progress when doing range testing.  The exception is from the `kable` library, but didn't quite match their docs.  

I'm going to remove the conditional logging and log the exception as a warning.  I think a BLE exception is a normal case in operation, but want to handle all exceptions (and hence the warning), since we should really not crash in this case, and the best we can do is log a warning message and continue without updating data.